script/optimize_images

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/perl
+
+# this script needs to be called periodically to reduce the size of the
+# stored images 
+
+use strict;
+use File::Basename;
+
+for (glob("/var/lib/openqa/images/*/*.unoptimized /var/lib/openqa/images/*/.thumbs/*.unoptimized")) {
```

Comment:
you could use a dedicated directory with symlinks to unoptimized files to avoid having to crawl all directories